docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -23,66 +23,67 @@ You don't need machine learning expertise to use Model Builder. All you need is
 
 You can bring many different scenarios to Model Builder, to generate a machine learning model for your application.
 
-A scenario is a description of the type of prediction you want to make on your data. For example:
+A scenario is a description of the type of prediction you want to make using your data. For example:
 - predict future product sales volume based on historical sales data
 - classify sentiments as positive or negative based on customer reviews
 - detect whether a banking transaction is fraudulent
 - route customer feedback issues to the correct team in your company
 
-In Model Builder, you need to map your scenario onto an [ML.NET task](resources/tasks.md). You can use Model Builder for **regression** (predicting numbers) and **classification** (predicting categories).
+## Choose a model type
 
-### Which machine learning scenario is right for me?
+In Model Builder, you need to select a machine learning model type. The type of model depends on what sort of prediction you are trying to make.
 
-Model Builder comes with templates for sentiment analysis, issue classification, price prediction, and custom scenarios. These templates can be used as a starting point for your specific ML.NET scenario.
+For scenarios that predict a number, the machine learning model type is called `regression`.
+
+For scenarios that predict a category, or put your data into different buckets, the model type is `classification`. There are two types of classification:
+- where there are only 2 categories, or buckets: `binary classification`.
```

Comment:
Suggestion: Use classes or groups instead of buckets

Reply:
I used buckets because that is the term I heard people using in the user studies. Do you think it sounds too weird?

Reply:
Have changed to categories, to trial that